usr/local/share/bastille/create.sh

```diff
@@ -58,7 +58,11 @@ validate_ip() {
           exit 1
         fi
       done
-      echo -e "${COLOR_GREEN}Valid: ($ip).${COLOR_RESET}"
+      if ifconfig | grep inet | grep -w "$ip" >/dev/null; then
```

Comment:
Can we just `grep -w "$ip"` without the second grep?

Reply:
Not sure if there's a case where the "IP address" is listed in any other line you don't want.
But I think it should be fine to remove the inet grep